incubator/binding-mqtt-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/kafka/streams/kafka/session.will.message.close.deliver.will.message/client.rpt

```diff
@@ -0,0 +1,344 @@
+write ${mqtt:will()
+            .topic("obituaries")
+            .delay(1)
+            .format("TEXT")
+            .payload("client-1 disconnected abruptly")
+            .lifetimeId("1")
+            .willId("1")
+      .build()}
+write flush
+
+write zilla:data.ext ${kafka:dataEx()
+                           .typeId(zilla:id("kafka"))
+                           .merged()
+                               .deferred(0)
+                               .partition(-1, -1)
+                               .key("client-1#will-signal")
+                               .hashKey("client-1")
+                               .header("type", "will-signal")
+                               .build()
+                           .build()}
+write ${mqtt:willSignal()
+            .clientId("client-1")
+            .delay(1)
+            .deliverAt(-1)
+            .lifetimeId("1")
+            .willId("1")
+        .build()}
+write flush
+
+write zilla:data.ext ${kafka:dataEx()
+                           .typeId(zilla:id("kafka"))
+                           .merged()
+                               .deferred(0)
+                               .partition(-1, -1)
+                               .key("client-1#will-signal")
+                               .hashKey("client-1")
+                               .header("type", "will-signal")
+                               .build()
+                           .build()}
+
+write ${mqtt:willSignal()
+            .clientId("client-1")
+            .delay(1)
+            .deliverAt(2)
+            .lifetimeId("1")
+            .willId("1")
+        .build()}
+write flush
+
+write close
+read closed
+
+
+connect await RECEIVED_WILL_DELIVER_AT_SIGNAL
+        "zilla://streams/kafka0"
+         option zilla:window 8192
+         option zilla:transmission "duplex"
+
+write zilla:begin.ext ${kafka:beginEx()
+                            .typeId(zilla:id("kafka"))
+                            .merged()
+                                .capabilities("FETCH_ONLY")
+                                .topic("mqtt_sessions")
+                                .filter()
+                                    .key("client-1#will")
+                                    .build()
+                                .build()
+                            .build()}
+
+connected
+
+read zilla:data.ext ${kafka:matchDataEx()
+                           .typeId(zilla:id("kafka"))
+                           .merged()
+                               .deferred(0)
+                               .partition(-1, -1)
+                               .key("client-1#will")
+                               .build()
+                           .build()}
+
+read ${mqtt:will()
+           .topic("obituaries")
+           .delay(1)
+           .format("TEXT")
+           .payload("client-1 disconnected abruptly")
+           .lifetimeId("1")
+           .willId("1")
+       .build()}
+
+write close
+read closed
+
+write notify WILL_MESSAGE_FETCHED
+
```

Comment:
Move `write notify` before `write close`, and change to `read notify` as it is after `read`, then rename to follow pattern, i.e. `SENT_SESSION_WILL_MESSAGE`.